src/fb.h

```diff
@@ -5,13 +5,23 @@
 
 #include "types.h"
 
-extern u32 *fb;
-extern int fb_s, fb_w, fb_h;
+typedef struct {
+    u32 *ptr;
+    int s;
+    int d;
+    int w;
+    int h;
+} fb_t;
```

Comment:
This needs to use real variable names and have a description as to what it's doing. Especially since s here is stride/4 which isn't actually the stride value anymore since it's not the number of bytes per line. It would probably be better to just keep track of the original bootargs.video information directly.

Reply:
I'll use better variables.